server/lib/user/index.js

```diff
@@ -8,6 +8,8 @@ const { getBySelector } = require('./user.getBySelector');
 const { getPicture } = require('./user.getPicture');
 const { getUserCount } = require('./user.getUserCount');
 const { getByTelegramUserId } = require('./user.getByTelegramUserId');
+const { getByNextcloudTalkToken } = require('./user.getByNextcloudTalkToken');
+const { getNextcloudTalkTokens } = require('./user.getNextcloudTalkTokens');
```

Comment:
Not we want Nextcloud as a part of Gladys core.
It should be a service, not changing Gladys core.